statisticsapp/src/main/java/com/example/statisticsapp/presentation/core/Event.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.statisticsapp.presentation.core
+
+class Event<out T>(private val content: T) {
```

Comment:
you can move event to core package or to domain (but probably core or ui-core will be better)